README.md

```diff
@@ -1,16 +1,30 @@
-# proiect-ip
-Proiect la Ingineria Programării \
+# Proiect la Ingineria Programării
+
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/44ab142848964c028d383f22cee1fac8)](https://www.codacy.com/gh/vlad-iancu/proiect-ip/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=vlad-iancu/proiect-ip&amp;utm_campaign=Badge_Grade)
 [![Codacy Badge](https://app.codacy.com/project/badge/Coverage/44ab142848964c028d383f22cee1fac8)](https://www.codacy.com/gh/vlad-iancu/proiect-ip/dashboard?utm_source=github.com&utm_medium=referral&utm_content=vlad-iancu/proiect-ip&utm_campaign=Badge_Coverage)
 
 ## Installation
-Create a virtual environment `python3 -m venv ./venv` \
-`source ./venv/bin/activate` \
-`pip3 install -r requirements.txt` \
-Create the `conf.json` file (see [conf.sample.json](./conf.sample.json)) \
-`./run.sh` \
-or \
-`run.ps1` if you are on Windows
+
+### Create a virtual environment, activate it, and install the dependencies
+
+```bash
+python3 -m venv ./venv
+source ./venv/bin/activate
```

Comment:
please also add the alternative Powershell command for activating the venv `./venv/Scripts/Activate.ps1`